BUILD

```diff
@@ -26,9 +23,6 @@ cc_binary(
 
 cc_binary(
     name = "libvsomeip3-e2e.so.3",
-    srcs = [
-        "libvsomeip3.so",
-    ],
     linkopts = [
         "-Wl,-rpath=\\$$ORIGIN:\\$$ORIGIN/../implementation:\\$$ORIGIN/../../implementation",
```

Comment:
this also has the same observation that maybe `\\$$ORIGIN` is enough. Did you test it?

Reply:
I did - e2e is not used in the test though. I removed it there too, same argument